pkg/version/version.go

```diff
@@ -42,7 +42,9 @@ func Print(program string) (string, error) {
 
 	var buf bytes.Buffer
 	tmpl := template.Must(template.New("version").Parse(versionInfoTmpl))
-	tmpl.ExecuteTemplate(&buf, "version", data)
+	if err := tmpl.ExecuteTemplate(&buf, "version", data); err != nil {
```

Comment:
Can you please keep this file aligned with the configuration from my other repositories, e.g. https://github.com/ricoberger/echoserver/blob/main/pkg/version/version.go